deploy/README.md

```diff
@@ -1,27 +1,121 @@
 # Overview of Infrastructure deployment and configuration
 
-The deployment involves the following steps outlined below:
+We are using bicep templates to deploy the infrastructure resources.
+We will be using following azure services:
+- Azure Batch Account
+- Azure Synapse Analytics
+- Azure Keyvault
+- Azure Storage Account
+- Azure Networking
+- Azure Log Analytics
 
-No | Step | Duration (approx.) | Required / Optional
----|------|----------|---------------------
-1 | [Preparing to execute the script](./deploy-infrastructure.md#preparing-to-execute-the-script) | 1 minute | required
-2 | [Infrastructure Deployment](./deploy-infrastructure.md#infrastructure-deployment) | 10 minutes | required
-3 | [Verifying infrastructure resources](./deploy-infrastructure.md#verifying-infrastructure-resources) | 5 minutes | required
+# Prerequisites
 
-Follow the [document](./deploy-infrastructure.md) to understand and execute the infrastructure deployment process.
+The deployment script uses following tools, please follow the links provided to install the suggested tools on your computer using which you would execute the script.
 
-## Executing the synapse pipeline
+- [git](https://github.com/git-guides/install-git) 
+- [az cli](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [jq](https://stedolan.github.io/jq/download/)
+- The scripts are executed on bash shell, so if using a computer with windows based operating system, install a [WSL](https://docs.microsoft.com/windows/wsl/about) environment to execute the script.
+- Install bicep on az cli using command `az bicep install`.
+- The bicep templates have been written to adhere to the syntax and rules for bicep version >= 0.8.9. Please check your bicep version using `az bicep version` if you run into bicep related errors.
+- The user performing the deployment of the bicep template and the associated scripts should have `Owner` role assigned at the subscription to which the resources are being deployed.
+- This solution assumes no interference from Policies deployed to your tenant preventing resources from being deployed. 
```

Comment:
we decided to move this line to troubleshooting.md and update as `TODO` so later we can add more details